Makefile

```diff
@@ -87,13 +87,13 @@ test:	rclone test_all
 
 # Quick test
 quicktest:
-	RCLONE_CONFIG="/notfound" go test $(BUILDTAGS) ./...
+	RCLONE_CONFIG="/notfound" go test -v $(BUILDTAGS) ./...
```

Comment:
I'm not in favour of adding the `-v` here. Rclone has a lot of tests and having to look through the `-v` output in the CI is not fun! (279 lines vs 68057) So I'm not in favour of making it longer.

Reply:
That's totally fair! It's not too hard to temporarily add the `-v` as needed. Done.